src/Packetery/Module/Options/Page.php

```diff
@@ -378,6 +383,20 @@ public function render(): void {
 			],
 			get_admin_url( null, 'admin.php' )
 		);
+		$latteParams['activeTab']            = ( $this->httpRequest->getQuery( self::PARAM_TAB ) ?? self::TAB_GENERAL );
+		$latteParams['generalTabLink']       = add_query_arg(
```

Comment:
Pro generování odkazu pro tab bych udělal metodu, které předáš parametr nepovinný parametr `tab`.  Odkazy na jednotlivé taby se budou lišit pouze v tomto parametru.